src/app/data_harvester.rs

```diff
@@ -100,7 +100,6 @@ impl Data {
 pub struct DataCollector {
     pub data: Data,
     sys: System,
-    mem_total_kb: u64,
```

Comment:
removing this since it's probably "better" in a sense to check it on every refresh period, rather than only once at the start. I guess one use case is some VM system with dynamic RAM?

In any case, the overhead is likely low, and zero if we already are getting the RAM usage.